__tests__/app.test.js

```diff
@@ -145,6 +145,31 @@ describe('App', () => {
         });
       })
     });
+
+    test('when a topic query is added, should filter the articles by the topic value specified in the query and return only articles with this topic', () => {
```

Comment:
Would be nice to see a test for an existing topic with no articles (I think 'paper' works here)